packages/components/src/NewDataGrid/NewDataGrid.test.tsx

```diff
@@ -721,4 +721,68 @@ describe('NewDataGrid', () => {
 
     expect(label).not.toBeInTheDocument();
   });
+
+  it('Предыдущие данные отображаются при isLoading=true', async () => {
+    type DataType = {
+      id: string;
+      documentName: string;
+    };
+
+    const pageData: DataGridRowWithOptions<DataType>[] = [
+      {
+        id: '1',
+        documentName: 'Договор №1',
+      },
+      {
+        id: '2',
+        documentName: 'Договор №2',
+      },
+    ];
+
+    const newPageData: DataGridRowWithOptions<DataType>[] = [
+      {
+        id: '3',
+        documentName: 'Договор №3',
+      },
+      {
+        id: '4',
+        documentName: 'Договор №4',
+      },
+    ];
+
+    const { rerender } = renderWithTheme(
+      <NewDataGrid
+        keyId="id"
+        columns={[
+          {
+            field: 'documentName',
+            label: 'Наименование',
+          },
+        ]}
+        rows={pageData}
+        onRetry={() => {}}
+      />,
+    );
+
+    rerender(
+      <NewDataGrid
+        keyId="id"
+        columns={[
+          {
+            field: 'documentName',
+            label: 'Наименование',
+          },
+        ]}
+        rows={newPageData}
```

Comment:
```suggestion
        rows={[]}
```

Когда идет загрузка новых данных нет + просто лишний код